### DISCUSSION TITLE
Cross Compile libsodium for x86 android

### DISCUSSION THREAD

**Discussion by rtvkiz** [jedisct1/libsodium#1057] [Q&A]
I seem to find no resources or examples to define how we can cross-compile libsodium library for x86 android devices. There is the documentation for ARM but need to know the parameters for x86.

**Comment by jedisct1** [comment]
Just run one of the `./dist-build/android-*` scripts.

**Comment by jedisct1** [comment]
For Android x86: `./dist-build/android-x86.sh` and x86_64: `./dist-build/android-x86_64.sh`.
